pipeline/samples/bazel/haskell/BUILD

```diff
@@ -1,12 +1,12 @@
 load(
     "@io_tweag_rules_haskell//haskell:haskell.bzl",
     "haskell_binary",
-    "haskell_import",
     "haskell_library",
+    "haskell_toolchain_library",
 )
 load("@ai_formation_hazel//:hazel.bzl", "hazel_library")
 
-haskell_import(name = "base")
+haskell_toolchain_library(name = "base")
```

Comment:
We should just delete the samples at this point but that is probably better done in a separate PR.

Reply:
Oh sorry, I didn’t realize this wasn’t merged yet when I made the PR to remove the samples. But it should hopefully be easy to fix.

Reply:
No worries, rebase was just a `git rm`.